.github/workflows/tests-outerloop.yml

```
@@ -61,42 +61,80 @@ jobs:
         run: |
           ${{ matrix.tests.command }}
 
+      - name: Upload logs, and test results
+        if: always()
+        uses: actions/upload-artifact@4cec3d8aa04e39d1a68397de0c4cd6fb9dce8ec1 # v4.6.1
+        with:
+          name: logs-${{ matrix.tests.project }}-${{ matrix.tests.os }}
+          path: |
+            ${{ github.workspace }}/artifacts/log/*/*.binlog
+            ${{ github.workspace }}/artifacts/log/*/TestLogs/**
+            ${{ github.workspace }}/artifacts/TestResults/*/*.trx
+          retention-days: 5
```

```suggestion
      - name: Upload binlogs
        if: failure()
        uses: actions/upload-artifact@4cec3d8aa04e39d1a68397de0c4cd6fb9dce8ec1 # v4.6.1
        with:
          name: ${{ matrix.tests.project }}-${{ matrix.tests.os }}-binlogs
          path: |
            ${{ github.workspace }}/artifacts/log/*/*.binlog
          retention-days: 5
```


After staring at workflow results like [this](https://github.com/RussKie/aspire/actions/runs/14483605243), I got to a realisation that artifacts should start with project name rather than something else (such as "logs-", "trx-", etc.) because it becomes too difficult to find all related artifacts.

---

How about `logs-{projectname}-{os}`? that makes it easier to spot them quickly, and differentiates it from any other artifacts we might upload.

---

I found that very confusing when looking through 200+ artifacts...

---

`logs-Aspire.Hosting.Tests-ubuntu-latest` vs `Aspire.Hosting.Tests-ubuntu-latest-logs`?
Sure, we can do that.